### PR TITLE
Add responsive meta tag and update layout to SLIM.

### PR DESCRIPTION
**Pull request by dinocosta**

### CODE REVIEW

File: app/views/layouts/application.slim [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+doctype html
+html
+  head
+    meta content=("text/html; charset=UTF-8") http-equiv="Content-Type"
+    meta name="viewport" content="width=device-width, initial-scale=1"
+    title
+      | SEI'#{@edition.id.to_s.byteslice(2,4)} :: Intranet
```
Comment (frm) [inline app/views/layouts/application.slim]: This should really just be a helper method.

File: app/views/layouts/application.slim [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```
Comment (frm) [inline app/views/layouts/application.slim]: You can remove this, can't you?

File: app/views/layouts/application.slim [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    title
+      | SEI'#{@edition.id.to_s.byteslice(2,4)} :: Intranet
+    = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true
+    = javascript_include_tag 'application', 'data-turbolinks-track' => true
+    = favicon_link_tag 'favicon.ico'
+    = csrf_meta_tags
+  body
+    = render 'layouts/navbar' if user_signed_in?
+    = yield
+
```
Comment (frm) [inline app/views/layouts/application.slim]: Unnecessary newline

**Review by frm — CHANGES_REQUESTED** [review]

File: app/views/layouts/application.slim (hunk 2 above)
Comment (dinocosta) [reply app/views/layouts/application.slim]: I think, according to [slim website](http://slim-lang.com), that the correct way to do this is `doctype html`.

File: app/views/layouts/application.slim (hunk 1 above)
Comment (dinocosta) [reply app/views/layouts/application.slim]: Noted 👍

File: app/views/layouts/application.slim (hunk 3 above)
Comment (dinocosta) [reply app/views/layouts/application.slim]: Getting picky are we? 😆

File: app/views/layouts/application.slim (hunk 2 above)
Comment (dinocosta) [reply app/views/layouts/application.slim]: @frmendes Forget what I said, did not check what was on line 2...

**Comment by dinocosta** [comment]
@frmendes Updated.

**Review by frm — APPROVED** [review]

**Pull request merged**